src/Sarif/Writers/SarifLogger.cs

```diff
@@ -278,11 +278,6 @@ private void CaptureFilesInResult(Result result)
             {
                 foreach (Location location in result.Locations)
                 {
-                    if (location.AnalysisTarget != null)
```

Comment:
Don't we want to "capture" `result.AnalysisTarget` if it exists? Perhaps _that's_ why the file contents were missing.

See, now I have three different theories about that :-)